docs/features/dead_letter_archive.md

```diff
@@ -98,3 +98,48 @@ Step Function Execution Status Change (detail) [here](https://docs.aws.amazon.co
     redriveStatusReason: [string],
 }
 ```
+
+## Search and View Dead Letter Archive Messages
+
+[Amazon Athena](https://docs.aws.amazon.com/athena/latest/ug/what-is.html) is a powerful serverless query service that allows us
+to analyze data directly from Amazon S3 using standard SQL. One of the key features of Athena is its support for partition
+projection. [Partition projection](https://docs.aws.amazon.com/athena/latest/ug/partition-projection.html) allows us to define a
+virtual partitioning scheme for our data stored in Amazon S3 without physically partitioning the data.
+
+We have provided an AWS Glue Catalog database, an AWS Glue Catalog table and an example query for querying S3 DLA messages.
+Our AWS Glue Catalog table `<prefix>_dla_glue_table` defines partition projection for `eventdate` key which corresponds
+to `date` folder under Dead Letter Archive S3 storage location.
+
+**Note:** `<prefix>` is your stack name with dash replaced by underscore
+
+### Procedure
+
+1. Navigate to AWS Athena Console:
+
+    Launch query editor to `Query your data with Trino SQL`.
+
+    Choose Workgroup `<prefix>_athena_workgroup` and acknowledge `Workgroup <prefix>_athena_workgroup settings`.
+
+    The `Saved queries` tab should have an example query `<prefix>_athena_test_query`, click it to open and run the query.
+
```

Comment:
in order for this to work, I also had to select my database (ecarton_ci_tf_glue_database) from the dropdown on the left. I assume this is expected and should be added to the docs